### PR TITLE
Day 2 angular

### PR DESCRIPTION
**Pull request by sujit-thinkitive**

### CODE REVIEW

File: day-1/hellow-world/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
```
Comment (yogesh-think) [inline day-1/hellow-world/.browserslistrc:4]: Don't check-in settings file. Use gitignore file to un track this file.

File: day-1/hellow-world/.browserslistrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+last 1 Chrome version
```
Comment (yogesh-think) [inline day-1/hellow-world/.browserslistrc:11]: Don't check-in settings file. Use gitignore file to un track this file.

File: day-1/hellow-world/.browserslistrc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+last 1 Firefox version
+last 2 Edge major versions
```
Comment (yogesh-think) [inline day-1/hellow-world/.browserslistrc:13]: Don't check-in settings file. Use gitignore file to un track this file.

File: day-1/hellow-world/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
```
Comment (yogesh-think) [inline day-1/hellow-world/.editorconfig:8]: Don't check-in settings file. Use gitignore file to un track this file.

File: day-2/TraningDay2/src/app/sign-in/user.validators.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import { AbstractControl, ValidationErrors } from "@angular/forms";
+
+
+export class UserValidators{
+    static usernameexists(control:AbstractControl): Promise<ValidationErrors | null> {
```
Comment (yogesh-think) [inline day-2/TraningDay2/src/app/sign-in/user.validators.ts:5]: Use Camel case naming convention

File: day-1/hellow-world/.editorconfig (hunk 4 above)
Comment (sujit-thinkitive) [reply day-1/hellow-world/.editorconfig:8]: Added gitignore file.

File: day-2/TraningDay2/src/app/sign-in/user.validators.ts (hunk 5 above)
Comment (sujit-thinkitive) [reply day-2/TraningDay2/src/app/sign-in/user.validators.ts:5]: Added gitignore file

File: day-1/hellow-world/.browserslistrc (hunk 3 above)
Comment (sujit-thinkitive) [reply day-1/hellow-world/.browserslistrc:13]: Added gitignore file

File: day-1/hellow-world/.browserslistrc (hunk 2 above)
Comment (sujit-thinkitive) [reply day-1/hellow-world/.browserslistrc:11]: Added gitignore file

File: day-1/hellow-world/.browserslistrc (hunk 1 above)
Comment (sujit-thinkitive) [reply day-1/hellow-world/.browserslistrc:4]: Added gitignore file

**Pull request merged**